cmd/device_info_linux.go

```diff
@@ -38,3 +39,29 @@ func getMac(iface string) (string, error) {
 	}
 	return strings.TrimSpace(string(macForIface)), nil
 }
+
+func getOSVersion() (string, error) {
+	osInfoPath := "/etc/os-release"
```

Comment:
Make a package level const. It likely compiles the same, but it's better stylistically.